schedule/kernel/bpftrace.yaml

```diff
@@ -0,0 +1,7 @@
+name:          bpftrace
```

Comment:
I think to call it just `bpf` would be better.

Reply:
Or bpftools because we have bpf in LTP and self tests as well

Reply:
`bpftools` sounds better than `bpf`